deploy/cache/base/deployment.yaml

```diff
@@ -51,6 +51,10 @@ spec:
               value: https://maven.repository.redhat.com/ga
             - name: STORE_REDHAT_TYPE
               value: maven2
+            - name: STORE_JSWEET_URL
```

Comment:
This file has no effect any more, it was still around for the transtion  but has just been deleted.

Reply:
Removed from this PR.